notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/InstallationTemplate.java

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
```

Comment:
Do we need to include this class in the preview?

Reply:
It doesn't hurt to leave it in, just that it doesn't actually do anything at this point

Reply:
Fixed in 468436b33819e131b38803e32a57be9f5abb65e6